docs/REST API/rest-api.md

```diff
@@ -1970,6 +1970,7 @@ the `software` table.
 | mdm_id                  | integer | query | The ID of the _mobile device management_ (MDM) solution to filter hosts by (that is, filter hosts that use a specific MDM provider and URL).                                                                                                                                                                                                |
 | mdm_name                | string  | query | The name of the _mobile device management_ (MDM) solution to filter hosts by (that is, filter hosts that use a specific MDM provider).                                                                                                                                                                                                |
 | mdm_enrollment_status   | string  | query | The _mobile device management_ (MDM) enrollment status to filter hosts by. Valid options are 'manual', 'automatic', 'enrolled', 'pending', or 'unenrolled'.                                                                                                                                                                                                             |
+| connected_to_fleet   | string  | query | Filter hosts that are talking to this Fleet server for MDM features. In rare cases, hosts can be enrolled to one Fleet server but talk to a different Fleet server for MDM features. In this case, the value would be `false`. Always `false` for Linux hosts.                                                                                                                           |
```

Comment:
I don't know if this is something you want to mention, but our main motivation was the opposite: osquery might report the host as talking to another MDM (ie: enrolled in the MDM sense of the word) but it's actually talking to Fleet under the hood (seamless migration)

This case is also true. Other than that looks good

Reply:
> our main motivation was the opposite: osquery might report the host as talking to another MDM (ie: enrolled in the MDM sense of the word) but it's actually talking to Fleet under the hood (seamless migration)

That's right.

I was thinking to myself what's easier to understand as someone reading the docs? The scenario I documented or this one?

I think it's probably the scenario I documented but I don't have a strong opinion.

I think the important piece to mention is that it's a rare scenario so the user reading the docs can be like "oh, that's interesting" and then forget about it.

Reply:
sounds good to me! thanks for the explanation